### DISCUSSION TITLE
findMany not returning relationships

### DISCUSSION THREAD

**Discussion by ghost** [keystonejs/keystone#7517] [Questions]
I am having a really strange error crop up.

Running
  ```js
const query = await context.db.Nonce.findMany({
    take: 5,
    query: `
    id
    nonce_id
    project {
      id
    }`,
  });
```

returns the full object, with the projectId, however
  ```js
const query = await context.query.Nonce.findMany({
    take: 5,
    query: `
    id
    nonce_id
    project {
      id
    }`,
  });
```
returns null for project.

In graphql playground:
```gql
query{
  nonces{
    id
    nonce_id
    project {
      id
    }
  }
}
```

Returns the data correctly.

It's only context.query that is not returning the related projects.

**Comment by borisno2** [comment]
Hi @kennedybaird 
Are you able to provide a simple reproduction (ie a repo that we can access) in order for us to give guidance?
